Code/Network/RKManagedObjectRequestOperation.m

```diff
@@ -849,17 +852,27 @@ - (BOOL)saveContext:(NSManagedObjectContext *)context error:(NSError **)error
 
 - (BOOL)saveContext:(NSError **)error
 {
-    if (self.willSaveMappingContextBlock) {
-        self.mappingResult = _responseMapperOperation.mappingResult;
-        [self.privateContext performBlockAndWait:^{
-            self.willSaveMappingContextBlock(self.privateContext);
-        }];
-    }
+    
+    // FIXME: Why was this only getting set `if (self.willSaveMappingContextBlock)` before?
+    self.mappingResult = _responseMapperOperation.mappingResult;
```

Comment:
@blakewatters @segiddins Do you know why we were setting the mapping result iff they provided us a block?